src/display/api.js

```diff
@@ -1088,16 +1088,24 @@ var PDFWorker = (function PDFWorkerClosure() {
   var nextFakeWorkerId = 0;
 
   function getWorkerSrc() {
+    var suffix;
```

Comment:
I declared those libraries in the top of the scope to avoid hoisting issues, if I just declare them when they are used Javascript it can be confusing.